Implementation/src/Logger/Log.cpp

```diff
@@ -4,6 +4,8 @@
 #include <iostream>
 // #include <string>
 
+#define CERRlOGS 1
```

Comment:
Should be set to one only during debug purposes, change it to 0 before pushing it